app/controllers/admin/themes_controller.rb

```diff
@@ -311,6 +311,22 @@ def export
     exporter.cleanup!
   end
 
+  def get_translations
+    raise Discourse::InvalidParameters.new(:locale) unless params[:locale]
+
+    I18n.locale = params[:locale]
```

Comment:
We should check here that the `locale` from the parameters is in `I18n.available_locales`. If not, we should throw an invalid param error early. 

It would also be useful to add a request spec for this controller, in `spec/requests/admin/themes_controller_spec.rb`.